src/components.rs

```diff
@@ -0,0 +1,16 @@
+use bevy::prelude::*;
+
+#[derive(Component, Debug, Clone, Copy, PartialEq)]
+pub struct Velocity {
+    pub v2: Vec2,
+}
```

Comment:
If the struct only contains one item we can use:
```suggestion
pub struct Velocity (Vec2);
```

Which would make the usage a little cleaner (we wouldn't need to access the `.v2` field)